paasta_tools/cli/cmds/check.py

```diff
@@ -139,7 +139,7 @@ def makefile_has_a_tab(makefile_path):
 
 def makefile_has_docker_tag(makefile_path):
     contents = get_file_contents(makefile_path)
-    return re.search(r"^DOCKER_TAG\s*\?=", contents, re.MULTILINE) is not None
+    return re.search(r"DOCKER_TAG\s*\?=", contents, re.MULTILINE) is not None
```

Comment:
Definitely an improvement. Do you think we should make this even less strict and not require `?=` to be used?